python/test_url.py

```
@@ -0,0 +1,90 @@
+        content = f.read()
+        matches = re.findall("[\'\"](https?://.*?)[\'\"]", content)
+        return matches
+
+
+def check_once(url):
+    try:
+        headers = {
+            'User-Agent':
+            'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.102 Safari/537.36'
+        }
+        response = requests.get(url, headers=headers)
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)